hack/build/bazel-build-builder.sh

```diff
@@ -28,13 +40,19 @@ if ! git diff-index --quiet HEAD~1 hack/build/docker; then
     echo "$DOCKER_PREFIX:$DOCKER_TAG"
 
     #Build the encapsulated compile and test container
-    (cd ${BUILDER_SPEC} && docker build --tag ${UNTAGGED_BUILDER_IMAGE}:${BUILDER_TAG} .)
+    if [ "${CDI_CONTAINER_BUILDCMD}" = "$(which buildah)" ]; then
+        (cd ${BUILDER_SPEC} && buildah build ${BUILDAH_PLATFORM_FLAG} --manifest ${UNTAGGED_BUILDER_IMAGE}:${BUILDER_TAG} .)
```

Comment:
`${UNTAGGED_BUILDER_IMAGE}:${BUILDER_TAG}` returns a lot, maybe worth to declare env var that will be equal to it and present the full image name ?